cms/templates/js/staff-lock-editor.underscore

```diff
@@ -7,26 +7,20 @@
     <% } %>
 </h3>
 <div class="modal-section-content staff-lock">
-    <ul class="list-fields list-input">
-        <li class="field field-checkbox checkbox-cosmetic">
-            <input type="checkbox" id="staff_lock" name="staff_lock" class="input input-checkbox" />
-            <label for="staff_lock" class="label">
-              <span class="icon fa fa-check-square-o input-checkbox-checked" aria-hidden="true"></span>
-              <span class="icon fa fa-square-o input-checkbox-unchecked" aria-hidden="true"></span>
-              <%- gettext('Hide from learners') %>
-            </label>
+    <input type="checkbox" id="staff_lock" name="staff_lock" class="input input-checkbox" />
```

Comment:
Nit: if you put the label tag around the input, you don't need to link them together with "for" and "id" values.